CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # Change Log
 
+## 1.1.2 - 2017-01-16
+
+### Fixed
+
+- [#28], Fixed a miscellaneuous bug with Fish 2.0.0
```

Comment:
I think "Fixed a miscellaneuous bug" is not really informational. It's basically not different from commit messages like "fix" or "misc."

How about saying "Follow-up to support the version of fish included in Ubuntu 14.04 LTS in config.fish." ?